app/controllers/users/merge_user.js

```diff
@@ -0,0 +1,37 @@
+import Ember from "ember";
+import AsyncMixin, { ERROR_STRATEGIES } from "stock/mixins/async";
+
+export default Ember.Controller.extend(AsyncMixin, {
+  user: Ember.computed.alias("model.user"),
+  otherUser: Ember.computed.alias("model.otherUser"),
+  userService: Ember.inject.service(),
+
+  masterUserId: Ember.computed("user.id", {
+    get() {
+      return this.get("user.id");
+    },
+    set(_, value) {
+      return value;
+    }
+  }),
```

Comment:
```suggestion
  masterUserId: Ember.computed.alias("user.id"),
```

Reply:
Since this property is used in radio-input, added proper getter and setter methods.
Setting up alias throws error `Assertion Failed: A record's id cannot be changed once it is in the loaded state` as its tries to update user's id.